src/Utility/Common.php

```diff
@@ -0,0 +1,160 @@
+<?php
```

Comment:
From where did you copy this file? There is already a DeepCopy implementation added in Drupal core. https://api.drupal.org/api/drupal/vendor%21myclabs%21deep-copy%21src%21DeepCopy%21DeepCopy.php/function/DeepCopy%3A%3AcopyArray/8.8.x